app/models/cdx/question.rb

```diff
@@ -1,16 +1,17 @@
 class CDX::Question < CDX::LoggedRequest
-  alias :get :perform
-
   private
 
   def request
-    client.call(:get_question, {
-      message: {
-        :securityToken => opts[:token],
-        :activityId =>    opts[:activity_id],
-        :userId =>        opts[:user][:UserId]
+    client.call(
+      :get_question,
+      {
+        message: {
+          securityToken: opts[:token],
+          activityId: opts[:activity_id],
+          userId: opts[:user][:UserId]
```

Comment:
`get_question` is a method we are passing to the SOAP client, which hits CDX. I believe CDX requires camelcase args since the XML keys are camelcase.